models.js

```diff
@@ -54,3 +55,25 @@ exports.selectCommentsByArticleId = (req) => {
       }
     });
 };
+
+exports.insertCommentByArticleId = (req) => {
+  const commentInsertSQL = format(
+    `INSERT INTO comments (body, author, article_id) VALUES %L RETURNING *;`,
+    [[req.body.body, req.body.username, req.params.article_id]]
+  );
+  const article_id = req.params.article_id;
+  const articleQuery = db.query(
```

Comment:
Do you need to make this query? if you just tried to add a comment to an article that didn't exist, would psql handle that for you?